src/components/TextLink/index.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Link } from 'react-router-dom';
```

Comment:
I don't think this component should handle any actual routing logic. This should be just a visual element - whatever routing library the user wants to use, its up to them.

Reply:
ok :) so just plain anchor